pkg/ebpf/c/common/consts.h

```diff
@@ -18,6 +18,7 @@
 #define SEND_META_SIZE      28
 
 #define MAX_MEM_DUMP_SIZE   127
+#define MAX_NUM_MODS        600       // number of iterations - value that the verifier was seen to cope with - the higher, the better.
```

Comment:
This macro name is not clear enough - does MODS imply modules?
Also - why should we put this under common? isn't it only used for your event?

Reply:
Changed it to MAX_NUM_MODULES.

Where do you recommend to put it?